Libraries/Components/TabBarIOS/TabBarIOS.android.js

```diff
@@ -12,13 +12,21 @@
 
 const React = require('React');
 const StyleSheet = require('StyleSheet');
-const TabBarItemIOS = require('TabBarItemIOS');
 const View = require('View');
 
-class DummyTabBarIOS extends React.Component<$FlowFixMeProps> {
-  static Item = TabBarItemIOS;
+let showedDeprecationWarning = false;
 
+class DummyTabBarIOS extends React.Component<$FlowFixMeProps> {
   render() {
+    if (!showedDeprecationWarning) {
```

Comment:
How do you feel about the different places this could go? At the top level, constructor, componentDidMount, render. My feeling is that I'd prefer constructor or componentDidMount but I'm curious if you had thought about this already.